src/panels/KeybindsPanel/KeybindsPanel.tsx

```diff
@@ -21,10 +21,10 @@ export function KeybindsPanel() {
         <Tabs.Panel value={'keyboardLayout'}>
           <KeyboardLayout />
         </Tabs.Panel>
-        <Tabs.Panel value={'listLayout'} h={windowHeight - tabsHeight}>
-          <ListLayout />
+        <Tabs.Panel value={'listLayout'}>
+          <ListLayout height={windowHeight - tabsHeight} />
```

Comment:
You can keep the original version here, list layout doesn't need to know the height